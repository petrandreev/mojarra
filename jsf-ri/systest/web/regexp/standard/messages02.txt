(?s).*<head>\s*<style\s*type="text/css"\s*media="screen">.*\.errors.*</style>\s*</head>\s*<body>\s*<table>\s*<tbody>\s*<tr>\s*<td>\s*<ul\s*dir="LTR"\s*style="left:\s*48px;\s*top:\s*100px;\s*position:\s*absolute">\s*<li\s*style="color:\s*yellow">\s*<span\s*title="Informational\s*Detail">\s*Information\s*Summary\s*Informational\s*Detail\s*</span>\s*</li>\s*<li>\s*<span\s*title="Warning\s*Detail">\s*Warning\s*Summary\s*Warning\s*Detail\s*</span>\s*</li>\s*<li\s*style="color:\s*red">\s*<span\s*title="Error\s*Detail">\s*Error\s*Summary\s*Error\s*Detail\s*</span>\s*</li>\s*<li\s*style="color:\s*blue">\s*<span\s*title="Fatal\s*Detail">\s*Fatal\s*Summary\s*Fatal\s*Detail\s*</span>\s*</li>\s*</ul>\s*</td>\s*</tr>\s*<tr>\s*<td>\s*<table\s*lang="en"\s*style="left:\s*48px;\s*top:\s*200px;\s*position:\s*absolute">\s*<tbody>\s*<tr\s*style="color:\s*yellow">\s*<td>\s*<span\s*title="Informational\s*Detail">\s*Information\s*Summary\s*Informational\s*Detail\s*</span>\s*</td>\s*</tr>\s*<tr>\s*<td>\s*<span\s*title="Warning\s*Detail">\s*Warning\s*Summary\s*Warning\s*Detail\s*</span>\s*</td>\s*</tr>\s*<tr\s*style="color:\s*red">\s*<td>\s*<span\s*title="Error\s*Detail">\s*Error\s*Summary\s*Error\s*Detail\s*</span>\s*</td>\s*</tr>\s*<tr\s*style="color:\s*blue">\s*<td>\s*<span\s*title="Fatal\s*Detail">\s*Fatal\s*Summary\s*Fatal\s*Detail\s*</span>\s*</td>\s*</tr>\s*</tbody>\s*</table>\s*</td>\s*</tr>\s*<tr>\s*<td>\s*<ul\s*class="errors">\s*<li>\s*Information\s*Summary\s*Informational\s*Detail\s*</li>\s*<li>\s*Warning\s*Summary\s*Warning\s*Detail\s*</li>\s*<li>\s*Error\s*Summary\s*Error\s*Detail\s*</li>\s*<li>\s*Fatal\s*Summary\s*Fatal\s*Detail\s*</li>\s*</ul>\s*</td>\s*</tr>\s*<tr>\s*<td>\s*<table\s*style="left:\s*48px;\s*top:\s*500px;\s*position:\s*absolute">\s*<tbody>\s*<tr\s*style="color:\s*yellow">\s*<td>\s*Information\s*Summary\s*Informational\s*Detail\s*</td>\s*</tr>\s*<tr>\s*<td>\s*Warning\s*Summary\s*Warning\s*Detail\s*</td>\s*</tr>\s*<tr\s*style="color:\s*red">\s*<td>\s*Error\s*Summary\s*Error\s*Detail\s*</td>\s*</tr>\s*<tr\s*style="color:\s*blue">\s*<td>\s*Fatal\s*Summary\s*Fatal\s*Detail\s*</td>\s*</tr>\s*</tbody>\s*</table>\s*</td>\s*</tr>\s*</tbody>\s*</table>\s*</body>.*
