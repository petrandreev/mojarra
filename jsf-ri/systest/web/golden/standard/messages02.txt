





                                                                                    

                                                                                    
<html>
<STYLE TYPE="text/css" MEDIA=screen>
<!--
.errors {
  background-color: #7171A5;
  border: 5px outset #71A5A5;
  border-collapse: collapse;
  font-family: sans-serif;
  font-size: 14pt;
  padding: 10px;
  left: 48px;
  top: 300px;
  position: absolute;
}
-->
</STYLE>


       
         

         

         

         
       <table>
<tbody>
<tr>
<td><ul dir="LTR" style="left: 48px; top: 100px; position: absolute"><li style="color: yellow"><span title="Information Summary">	Informational Detail</span></li><li><span title="Warning Summary">	Warning Detail</span></li><li style="color: red"><span title="Error Summary">	Error Detail</span></li><li style="color: blue"><span title="Fatal Summary">	Fatal Detail</span></li></ul></td>
</tr>
<tr>
<td><table lang="en" style="left: 48px; top: 200px; position: absolute"><tr style="color: yellow"><td><span title="Information Summary">	Informational Detail</span></td></tr><tr><td><span title="Warning Summary">	Warning Detail</span></td></tr><tr style="color: red"><td><span title="Error Summary">	Error Detail</span></td></tr><tr style="color: blue"><td><span title="Fatal Summary">	Fatal Detail</span></td></tr></table></td>
</tr>
<tr>
<td><ul class="errors"><li>	Information Summary Informational Detail</li><li>	Warning Summary Warning Detail</li><li>	Error Summary Error Detail</li><li>	Fatal Summary Fatal Detail</li></ul></td>
</tr>
<tr>
<td><table style="left: 48px; top: 500px; position: absolute"><tr style="color: yellow"><td>	Information Summary Informational Detail</td></tr><tr><td>	Warning Summary Warning Detail</td></tr><tr style="color: red"><td>	Error Summary Error Detail</td></tr><tr style="color: blue"><td>	Fatal Summary Fatal Detail</td></tr></table></td>
</tr>
</tbody>
</table>


</html>
